### PR TITLE
bump versions

### PR DESCRIPTION
**Pull request by coffeepac**
weekly version bump.  leaving the old style k8s version pinned to the 1.4 series.  it should be removed in the next two weeks.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -12,17 +12,17 @@ ENV     CLOUDSDK_PYTHON_SITEPACKAGES 1
...
-ENV     K8S_VERSION_1_4=v1.4.9
-ENV     K8S_VERSION_1_5=v1.5.4
-ENV     K8S_VERSION_1_6=v1.6.0-beta.4
+ENV     K8S_VERSION_1_4=v1.4.12
+ENV     K8S_VERSION_1_5=v1.5.7
+ENV     K8S_VERSION_1_6=v1.6.3
 
 ENV     K8S_HELM_VERSION_1_4=v2.1.3
 ENV     K8S_HELM_VERSION_1_5=v2.3.1
 # Currently, there is no support for K8s 1.6. Version 2.4.0 will be released May 1, 2017 and will support 1.6.
```
Comment (guineveresaenger) [inline Dockerfile]: can we get rid of the comment please? It no longer applies, and the problem is going to persist, but not with these numbers.

File: Dockerfile [hunk 2]
```diff
@@ -12,17 +12,17 @@ ENV     CLOUDSDK_PYTHON_SITEPACKAGES 1
...
-#ENV     K8S_HELM_VERSION_1_6=v2.4.0
+#ENV     K8S_HELM_VERSION_1_6=v2.4.1
```
Comment (guineveresaenger) [inline Dockerfile]: should be uncommented

File: Dockerfile [hunk 3]
```diff
@@ -12,17 +12,17 @@ ENV     CLOUDSDK_PYTHON_SITEPACKAGES 1
...
-ENV     K8S_VERSION=v1.4.4
+ENV     K8S_VERSION=v1.4.12
 ENV     K8S_HELM_VERSION=v2.3.1
 
-ENV     K8S_VERSION_1_4=v1.4.9
-ENV     K8S_VERSION_1_5=v1.5.4
-ENV     K8S_VERSION_1_6=v1.6.0-beta.4
+ENV     K8S_VERSION_1_4=v1.4.12
+ENV     K8S_VERSION_1_5=v1.5.7
+ENV     K8S_VERSION_1_6=v1.6.3
```
Comment (guineveresaenger) [inline Dockerfile]: 1.6.4 came out 5 days ago

Comment (coffeepac) [inline Dockerfile]: technically after this PR was submitted!  I can bump it though.

File: Dockerfile (hunk 2 above)
Comment (coffeepac) [inline Dockerfile]: yep.

File: Dockerfile (hunk 1 above)
Comment (coffeepac) [inline Dockerfile]: sure.

**Comment by guineveresaenger** [comment]
LGTM. Does k2-tools have a test? If so:
ok to test

**Pull request merged**